code_generation/templates/src/power_grid_model/core/metadata_enums.py.jinja

```diff
@@ -0,0 +1,72 @@
+# SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+"""Data types for power grid model dataset and component types."""
+
+# This file is automatically generated. DO NOT modify it manually!
+
+{% set dataset_types = all_map.keys() %}
+{% set components = all_map['input'].keys() %}
+
+from enum import Enum
+from typing import Any, Dict, Mapping
+
+# pylint: disable=invalid-name
```

Comment:
Why?

Reply:
I assume you are asking about `# pylint: disable=invalid-name`. Pylint doesn't like non-capitalized `Enum` keys, but I am following the pgm convention as in `src/power_grid_model/enum.py`.